typings/index.test-d.ts

```diff
@@ -665,6 +668,7 @@ client.on('interaction', async interaction => {
 
   const actionRow = new MessageActionRow({ components: [button] });
 
+  // @ts-expect-error
```

Comment:
What error are you expecting here?

Reply:
The error is you can't use the old `Message*` classes as components.

Reply:
I thought that was only deprecated though. Wouldn’t that make this semver major?

Reply:
No because this error only affects typescript, in this specific method it's hard to mark the method type as deprecated. But using the old classes here still work as expected.

Reply:
Can’t you simply deprecate the class/constructor, emit the deprecation warning and keep it working without the error? I know typescript breaking changes don’t really make things semver major but it would be nice if the error was avoided imo